lib/hyrax.rb

```diff
@@ -16,6 +16,7 @@
 require 'hyrax/engine'
 require 'hyrax/version'
 require 'hyrax/inflections'
+require 'hyrax/name'
```

Comment:
I'm unclear why this is added.

Reply:
it turned out that actually running this generator and starting an app caused `NameError` on this module. it seems like it should have just always been there.